.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build
+
+on:
+  pull_request:
+    types: [ labeled ]
+
+jobs:
+  build:
+    # only run a manual build when the PR comes from gatsby
+    # as gatsby doesn't run on automated pull requests
+    if: ${{ github.event.label.name == 'dependencies' }}
+    name: Build
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: Install
+        run: yarn
```

Comment:
I suggest we use `yarn install --frozen-lockfile` here (and in the other workflows). Since this workflow is especially sensible to dependency version changes, let's install the exact ones specified in the lock file.

Reply:
Yep, good idea, thanks 👍